docs/usage/configuration-options.md

```diff
@@ -185,6 +185,13 @@ Example use:
 }
 ```
 
+## automergeSchedule
+
+The `automergeSchedule` option allows you to define times of week or month for automerging Renovate updates.
+The default value for `automergeSchedule` is "at any time", which is functionally the same as declaring a `null` schedule.
+i.e. Renovate will auto merge any merge-able update requests available.
+To configure this option refer to [`schedule`](https://docs.renovatebot.com/configuration-options/#schedule) as the syntax is the same.
```

Comment:
```suggestion
To configure this option refer to [`schedule`](#schedule) as the syntax is the same.
```

Would this work as a link within the file itself?

Reply:
yep, it works